compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/XCLBinGen.cpp

```diff
@@ -1155,29 +1181,29 @@ LogicalResult aie2xclbin(
                  << errorMessage;
     return failure();
   }
-  for (auto w : *npuInstructions) output->os() << llvm::format("%08X\n", w);
+  for (uint32_t w : npuInstructions) output->os() << llvm::format("%08X\n", w);
   output->keep();
 
   Path unifiedObj = Path(tempDir) / "input.o";
   if (failed(generateUnifiedObject(
-          ctx, moduleOp, unifiedObj.string(), printIRBeforeAll, printIRAfterAll,
+          ctx, deviceOp, unifiedObj.string(), printIRBeforeAll, printIRAfterAll,
           printIRModuleScope, timing, useChess, verbose, tempDir, vitisDir,
           targetArch, peanoDir)))
-    return moduleOp.emitOpError("Failed to generate unified object");
+    return deviceOp.emitOpError("Failed to generate unified object");
 
-  if (failed(generateCoreElfFiles(moduleOp, unifiedObj.string(), tempDir,
+  if (failed(generateCoreElfFiles(deviceOp, unifiedObj.string(), tempDir,
                                   useChess, vitisDir, targetArch, verbose,
                                   peanoDir, ukernel)))
-    return moduleOp.emitOpError("Failed to generate core ELF file(s)");
+    return deviceOp.emitOpError("Failed to generate core ELF file(s)");
 
-  if (failed(generateCDO(ctx, moduleOp, printIRBeforeAll, printIRAfterAll,
+  if (failed(generateCDO(ctx, deviceOp, printIRBeforeAll, printIRAfterAll,
                          printIRModuleScope, timing, tempDir)))
-    return moduleOp.emitOpError("Failed to generate CDO");
+    return deviceOp.emitOpError("Failed to generate CDO");
 
   if (failed(generateXCLBin(outputXCLBin, tempDir, xclBinKernelID,
                             xclBinKernelName, xclBinInstanceName,
                             amdAIEInstallDir, verbose, InputXCLBin)))
-    return moduleOp.emitOpError("Failed to generate XCLBin");
+    return deviceOp.emitOpError("Failed to generate XCLBin");
```

Comment:
same conditional braces nit

Reply:
I've tried to address these, may have missed some though. Do you know if this is something we/I could use clang-tidy to help with?

Reply:
> I've tried to address these, may have missed some though. Do you know if this is something we/I could use clang-tidy to help with?

We probably could but clang-tidy isn't safe to just blanket apply like clang-format - I had something like that on mlir-aie back in the day (lol) but it led to unsafe/illegal edits.

Reply:
Anyway just leave whichever ones you don't want to bother with. We'll come back to em later (someone will lol...)